docs/resources/Channel.md

```diff
@@ -277,7 +317,7 @@ Edit a previously sent message. You can only edit messages that have been sent b
 
 Delete a message. If operating on a guild channel and trying to delete a message that was not sent by the current user, this endpoint requires the 'MANAGE_MESSAGES' permission. Returns a 204 empty response on success. Fires a [Message Delete](#DOCS_GATEWAY/message-delete) Gateway event.
 
-## Bulk Delete Messages % POST /channels/{channel.id#DOCS_CHANNEL/channel-objects}/messages/bulk_delete
+## Bulk Delete Messages % POST /channels/{channel.id#DOCS_CHANNEL/channel-objects}/messages/bulk-delete
```

Comment:
A lot of people wondered if the old endpoint is being removed. We're only deprecating it - (and it may be removed in API v7) - we should add a note?